<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDocumentWindow</class>
 <widget class="QWidget" name="NewDocumentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление документа</string>
  </property>
  <widget class="QPushButton" name="BtnReset">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>610</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_baseDoc">
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_patronymic">
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Номер документа</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Время выдачи документа</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox_otherDoc">
      <property name="text">
       <string>другой</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Название гос. учреждения, 
выдавший документ</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Пол</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QDateTimeEdit" name="time_start">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Версия документа</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Отчество</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Фамилия</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Время начало действия</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Время окончания действия</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_typeDoc">
      <property name="text">
       <string>Тип документа</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox_personInDB">
      <property name="text">
       <string>человека нет в БД</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QDateTimeEdit" name="time_issuing">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_id">
      <property name="text">
       <string>id человека</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDateTimeEdit" name="time_create">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="combo_sex">
      <item>
       <property name="text">
        <string>Мужской</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Женский</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QDateTimeEdit" name="time_end">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_idPerson">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_authority"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="combo_typeDoc">
      <item>
       <property name="text">
        <string>Обычный документ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Рождение</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Смерть</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Заключение брака</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Расторжение брака</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Работа</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Обучение</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Место проживания</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_surname"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Время создания документа</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_numDoc">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="spinBox_version">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QCheckBox" name="checkBox_noendDoc">
      <property name="text">
       <string>бессрочный</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newTypeDoc">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_otherName"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_otherValue"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Значение</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_addOtherAtribute">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>10</y>
     <width>31</width>
     <height>501</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>201</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>название поля</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>значение</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>31</width>
     <height>501</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>170</y>
     <width>311</width>
     <height>217</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_marriage">
    <item row="6" column="1">
     <widget class="QComboBox" name="combo_sexPartner">
      <item>
       <property name="text">
        <string>Мужской</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Женский</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_dateTimeMarStart">
      <property name="text">
       <string>Дата и время заключения</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Отчество супруга</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_partnerSurname"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Фамилия супруга</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_partnerPatronymic"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>id Супруга</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Имя супруга</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_marriageEnd">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_idPartner">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_marriageStart">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Пол супруга</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_partnerName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_dateTimeMarEnd">
      <property name="text">
       <string>Дата и время расторжения</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>296</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_work">
    <item row="5" column="0">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>Должность</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Время окончания работы</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>Место работы</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>Название организации</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>Зарплата</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Время начало работы</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_workPlace"/>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_workStart">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_workEnd">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_workOrganization"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_workPosition"/>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_workSalary">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="singleStep">
       <number>500</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>320</width>
     <height>147</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_study">
    <item row="3" column="0">
     <widget class="QLabel" name="label_49">
      <property name="text">
       <string>Время окончания учебы</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_50">
      <property name="text">
       <string>Место учебы</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_48">
      <property name="text">
       <string>Специальность</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_51">
      <property name="text">
       <string>Название учебного зведения</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_studyOrganization"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_studySpeciality"/>
    </item>
    <item row="2" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_studyStart">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_studyPlace"/>
    </item>
    <item row="3" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_studyEnd">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_53">
      <property name="text">
       <string>Время начало учебы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>991</x>
     <y>10</y>
     <width>242</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_birth">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Место рождения</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Дата и время
рождения</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_birthPlace"/>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_birth">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>80</y>
     <width>257</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_death">
    <item row="1" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Дата и время
смерти</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_deathPlace"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Место смерти</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_death">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Тип смерти</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="combo_deathType">
      <item>
       <property name="text">
        <string>Неизвестно</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сердечный приступ</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>395</y>
     <width>270</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_residence">
    <item row="0" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Тип проживания</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Название населенного
пункта</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>Дом</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Область/Край</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Подъезд</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Корпус/строение</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Тип населенного пункта</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Улица</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Квартира</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_resRegion"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_resLocality"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_resStreet"/>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_resHouse">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_resBuilding">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_resPorch">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spinBox_resFlat">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="combo_resType">
      <item>
       <property name="text">
        <string>Постоянное</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Временное</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="combo_resTypeLocality">
      <item>
       <property name="text">
        <string>Город</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Деревня</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Хутор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Поселок</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Село</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBox_personInDB</tabstop>
  <tabstop>combo_typeDoc</tabstop>
  <tabstop>checkBox_otherDoc</tabstop>
  <tabstop>lineEdit_surname</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_patronymic</tabstop>
  <tabstop>combo_sex</tabstop>
  <tabstop>time_create</tabstop>
  <tabstop>lineEdit_authority</tabstop>
  <tabstop>time_issuing</tabstop>
  <tabstop>time_start</tabstop>
  <tabstop>time_end</tabstop>
  <tabstop>lineEdit_otherName</tabstop>
  <tabstop>lineEdit_otherValue</tabstop>
  <tabstop>btn_addOtherAtribute</tabstop>
  <tabstop>lineEdit_birthPlace</tabstop>
  <tabstop>dateTimeEdit_birth</tabstop>
  <tabstop>lineEdit_deathPlace</tabstop>
  <tabstop>dateTimeEdit_death</tabstop>
  <tabstop>dateTimeEdit_marriageStart</tabstop>
  <tabstop>dateTimeEdit_marriageEnd</tabstop>
  <tabstop>lineEdit_partnerSurname</tabstop>
  <tabstop>lineEdit_partnerName</tabstop>
  <tabstop>lineEdit_partnerPatronymic</tabstop>
  <tabstop>lineEdit_resRegion</tabstop>
  <tabstop>lineEdit_resLocality</tabstop>
  <tabstop>lineEdit_resStreet</tabstop>
  <tabstop>spinBox_resHouse</tabstop>
  <tabstop>spinBox_resBuilding</tabstop>
  <tabstop>spinBox_resPorch</tabstop>
  <tabstop>spinBox_resFlat</tabstop>
  <tabstop>BtnReset</tabstop>
  <tabstop>BtnOk</tabstop>
  <tabstop>lineEdit_newTypeDoc</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
